crowdsec-docs/docs/intro.mdx

```diff
@@ -7,17 +7,25 @@ id: intro
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import AcademyPromo from '@site/src/components/AcademyPromo';
 
+<!-- @tko
+
+ - replace the image with a gif 
+ - edit the image to have the same background color
+ - add the cool miniature for the console
+
+-->
+
 <div style={{display: 'flex'}}>
     <div style={{textAlign: 'center', flex: '1'}}>
-    <img width="800" height="auto" src={useBaseUrl('/img/crowdsec_ecosystem.png')} />
+    <img width="800" height="auto" src={useBaseUrl('/img/simplified_SE_overview.png')} />
```

Comment:
- Changer le titre pour pas qu'on ait uniquement "Introduction" genre **"Security Engine Overview"**
- Et un peu brute de décoffrage de commencer la page sur un schéma sans meme un mini mot d'introduction.  
Proposition de phrase avant le schéma:

Here’s a quick overview of our FOSS IDS/IPS, the CrowdSec Security Engine—from core features to its architecture and setup suggestions tailored to your infrastructure.